src/utils.ts

```diff
@@ -53,14 +53,13 @@ export function adaptLog(msg: string): string {
 }
 
 const DOCKER_HOST = "host.docker.internal";
-const MACOS_PLATFORM = "darwin";
 /**
- * Adapts `url` by replacing localhost and 127.0.0.1 with `host.internal.docker`
+ * Adapts `url` by replacing localhost and 127.0.0.1 with `host.docker.internal`
  * @param url string representing the url to be adapted
  * @returns adapted url
  */
-export function adaptUrl(url: string): string {
-    if (process.platform === MACOS_PLATFORM) {
+export function adaptUrl(url: string, isDockerDesktop: boolean): string {
```

Comment:
Instead of having to provide the boolean argument, could this `adaptUrl` function find out by itself if the Docker is indeed a desktop one?

Reply:
i guess this logic was in utils.ts to make it accessible from anywhere, that's also why i moved isDockerDesktop function to utils.ts

Reply:
`adaptUrl` was in utils because it really was used in multiple places. Now it's used in only one place (after a refactor was done a few months ago).

Reply:
refactored